content/docs/rfcs/32/README.md

```diff
@@ -145,13 +149,13 @@ the user need to submit the following fields to the circuit:
 The proof output is calculated locally,
 in order for the required fields for proof verification to be sent along with the proof.
 The proof output is composed of the `y` share of the secret equation and the `internal_nullifier`.
-The `internal_nullifier` represents an unique fingerprint for the user for the `external_nullifier`.
+The `internal_nullifier` represents an unique fingerprint for the user for the `epoch`.
```

Comment:
I think here we still need to clarify that `internal_nullifier` depends on `rln_identifier` too. For example we can say:
```
The `internal_nullifier` represents a unique fingerprint of a user for the `epoch` within the app.
```